### DISCUSSION TITLE
I am having a Type error when using CustomEvent

### DISCUSSION THREAD

**Discussion by sojimaxi** [SAP/ui5-webcomponents-ngx#61] [Q&A]
I seem not to be able to get this to work properly

````html
  <ui5-shellbar  (ui5ProfileClick)="onProfileClick($event)">
    </ui5-avatar>
  </ui5-shellbar>
````

In component

````ts
  onProfileClick(event: { targetRef: HTMLElement }){
    this.profileMenu.showAt?.(event.targetRef)
  }

````

I get an error because the type of event should be `CustomEvent` and if I change it I will need to use `event.detail.targetRef` which is not defined.

**Comment by koeppel** [comment]
@sojimaxi is the issue still persistent?
